.gitignore

```
@@ -1,2 +1,5 @@
 .deploy
-.idea
\ No newline at end of file
+.idea
+
+./invenio/charts/*
+./invenio/values-*.yaml
```

```suggestion
/invenio/values-*.yaml
```

same as previous comment